Wordle.UI/ConsoleUI.cs

```diff
@@ -0,0 +1,124 @@
+using Wordle.Domain;
+namespace Wordle.UI;
+
+public class ConsoleUI : IWordleUI
+{
+    Game game = new Game();
+    GuessValidator guessValidator = new GuessValidator();
+
+    
+    public void TakeTurns(string answer, GuessStatistics guesses)
+    {
+        var guess = GetGuess();
+        var letterScoresList = game.EvaluateGuess(answer, guess);
+        guesses.UpdateGuessStatistics(letterScoresList);
+        UpdateBoard(guesses.GuessArray, guesses.GuessCount);
+    }
```

Comment:
there is no UI code in this UI method; which makes me believe it belongs somewhere else. Like "main" or `Game` maybe?

Reply:
Likewise, every UI would have to have its own algorithm for taking turns, which is something that should be defined only once. In other words, I would not expect the process for taking turns to change or be different for a console game vs. a web game vs. a phone app.

Reply:
I started with trying to add `TakeTurns` into `Game`, but the `GetGuess` and `UpdateBoard` board methods are UI implementation and Wordle.UI can't be accessed from Wordle.Domain. My thought process was that no matter the UI there should be some functionality that allows a player to take a turn, so it might belong in the interface as well. But I would be amicable to moving it into `Main`! That makes sense to me.